index.bs

```diff
@@ -328,9 +329,8 @@ more arguments are left. It returns a [=/list=] of objects suitable for printing
   1. <p class="XXX">TODO: process %c</p>
   1. If any of the previous steps set |converted|, replace |specifier| in |target| with
      |converted|.
-  1. Let |result| be a [=/list=] containing |target| together with the elements of |args| starting
-     from the third onward.
-1. If |target| does not have any format specifiers left, return |result|.
+1. Let |result| be a [=/list=] containing |target| together with the elements of |args| starting
+   from the third onward.
 1. If |result|'s [=list/size=] is 1, return |result|.
```

Comment:
Can we get rid of this line "1. If |result|'s [=list/size=] is 1, return |result|." now that we're handling the args.length == 1 case as the first step of Formatter?

Reply:
Thanks for looking at this. You're right, that's redundant now. I've removed it.